misc-scripts/proposal_vote.py

```diff
@@ -0,0 +1,109 @@
+from distutils.debug import DEBUG
+import os
+import argparse
+import re
+from tabnanny import check
```

Comment:
If external libraries are used make sure to include them in the `requirements.txt` file. In this case `tabnanny`. please make sure to run in venv(Virtual environment.) rather than running it on system level.